### ISSUE TITLE
New Beta?

### ISSUE THREAD

**Issue by Metaphorce812** [tvrenamer/tvrenamer#383]
I was just wondering if someone could compile a new beta since the latest release is 15 months old and it looks like there have been a lot of changes made to the program since then. Thanks to the devs for this great program!

**Comment by StephKoenig** [comment]
I agree, I'd love a release that encapsulates the developments since '17. It's such an awesome program that it seems a waste to have the efforts sat unused (and unappreciated!)

**Comment by vipuldelwadia** [comment]
1.0 is out, and this is covered: 1.0b5 and 1.0b6 shipped, and then 1.0 itself.

Release: https://github.com/tvrenamer/tvrenamer/releases/tag/v1.0

Closing. Please reopen if it is still a problem on 1.0.
